src/main/scala/org/tensorframes/impl/datatypes.scala

```diff
@@ -488,4 +558,69 @@ private[impl] object LongOperations extends ScalarTypeOperation[Long] with Loggi
     logTrace(s"Extracted from buffer: ${res.toSeq}")
     res
   }
-}
\ No newline at end of file
+
+  override def tag: Option[TypeTag[_]] = Option(implicitly[TypeTag[Long]])
+
+  override def ev = ClassTag.Long
+}
+
+// ********** STRING *********
+// This is actually byte arrays, which corresponds to the 'binary' type in Spark.
```

Comment:
what if you pass in a real string column?? wouldn't be cleaner to explicitly pass in bytearrays?